source/_integrations/backup.markdown

```diff
@@ -66,3 +66,7 @@ automation:
       alias: "Create backup now"
       service: backup.create
 ```
+
+## Restoring from a backup
+
+To restore from a backup, see [Restoring a Backup on a new install](/common-tasks/supervised/#restoring-a-backup-on-a-new-install)
```

Comment:
This is partially correct, but not was meant in the original issue/feedback raised.

The linked section is how to restore it onto an OS installation. It does not provide information on how to restore it onto a Container or Core installation type.

Reply:
@frenck I have added new pages for core and container backup restores. I used file paths used in the install guides so hopefully there is a bit of continuity.

I removed the addon folder from the directories backed up in core and container because I believe addons are only available for OS and supervised. I haven't verified this I went through the install to verify the folder paths but not to the point of running it. If you can confirm this then I hope all the other changes should address the feedback.